src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml

```diff
@@ -5,12 +5,16 @@
     xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
     <Grid>
         <esriUI:MapView x:Name="MyMapView" />
-        <Border Background="White" BorderBrush="Black" BorderThickness="1"
-                HorizontalAlignment="Right" VerticalAlignment="Top"
-                Margin="30" Padding="20" Width="375">
-            <TextBlock Text="Tap to identify features."
-                       TextAlignment="Center"
-                       FontWeight="SemiBold" />
+        <Border Style="{StaticResource BorderStyle}">
+            <StackPanel>
+                <TextBlock Text="Tap to identify features."
+                           TextAlignment="Center"
+                           FontWeight="SemiBold" />
+                <WebView x:Name="ResultWebView"
```

Comment:
We should probably update this sample's thumbnail image, which is still showing a callout.